src/permutationtable.rs

```diff
@@ -26,7 +26,7 @@ const TABLE_SIZE: usize = 256;
 ///
 /// Table creation is expensive, so in most circumstances you'll only want to
 /// create one of these per generator.
-#[allow(missing_copy_implementations)]
+#[derive(Copy)]
```

Comment:
I started a branch off of this that removes the Copy on PermutationTable [here](/Razaekel/noise-rs/tree/NoCopyPermutationTable). I haven't gone so far as to require only references on Select yet.